src/nyx/features/sharpness.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include <vector>
+#include "../helpers/helpers.h"
+#include "../roi_cache.h"
+#include "image_matrix.h"
+#include "../feature_method.h"
+#include "../environment.h"
+
+#define EPSILON 1e-8
```

Comment:
There is already an `EPSILON` variable in Nyxus somwhere. Use a `constexpr` instead of macro